pcsx2/Counters.cpp

```diff
@@ -270,6 +270,30 @@ static const char* ReportVideoMode()
 	}
 }
 
+Fixed100 GetVerticalFrequency()
+{
+	switch (gsVideoMode)
+	{
+	case GS_VideoMode::Uninitialized: // SYSCALL instruction hasn't executed yet, give some temporary values.
+		return 60;
+	case GS_VideoMode::PAL:
+		return EmuConfig.GS.FrameratePAL;
+	case GS_VideoMode::NTSC:
+		return EmuConfig.GS.FramerateNTSC;
+	case GS_VideoMode::HDTV_480P:
+		return 59.94;
+	case GS_VideoMode::HDTV_1080P:
+	case GS_VideoMode::HDTV_1080I:
```

Comment:
Since today is a special day, I'll explain it more thoroughly :P 

There are generally two types of frequency for a video mode, one is vertical and the other one is horizontal.

```
----.-----------.---------------.-----------.-----------.
 No | Name      | Resolution	| fV(Hz)    | fH(kHz)   |
----+-----------+---------------+-----------+-----------|
  0 | NTSC-NI   |  640x240(224) | 59.940    | 15.734    |
  1 | NTSC-I    |  640x480(448) | 59.820    | 15.734    |
  2 | PAL-NI    |  640x288(256) | 50.000    | 15.625    |
  3 | PAL-I     |  640x576(512) | 49.760    | 15.625    |
  4 | VESA-1A   |  640x480      | 59.940    | 31.469    |
  5 | VESA-1C   |  640x480      | 75.000    | 37.500    |
  6 | VESA-2B   |  800x600      | 60.317    | 37.879    |
  7 | VESA-2D   |  800x600      | 75.000    | 46.875    |
  8 | VESA-3B   | 1024x768      | 60.004    | 48.363    |
  9 | VESA-3D   | 1024x768      | 75.029    | 60.023    |
 10 | VESA-4A   | 1280x1024     | 60.020    | 63.981    |
 11 | VESA-4B   | 1280x1024     | 75.025    | 79.976    |
 12 | DTV-480P  |  720x480      | 59.940    | 31.469    |
 13 | DTV-1080I | 1920x1080     | 60.000    | 33.750    |
 14 | DTV-720P  | 1280x720      | ??        | ??        |
----^-----------^---------------^-----------^-----------'
```

Note that vertical frequency is _different_ from the [horizontal frequency](https://en.wikipedia.org/wiki/Horizontal_scan_rate) and frame frequency. (AKA FPS)

Here's an example from Wikipedia : 
> For example, most movie projectors advance from one frame to the next one 24 times each second. But each frame is illuminated two or three times before the next frame is projected using a shutter in front of its lamp. As a result, the movie projector runs at 24 frames per second, but has a 48 or 72 Hz refresh rate.

So in our case,

NTSC -> Vertical frequency of 59.94Hz and a frame frequency of 29.97 fps
PAL -> Vertical frequency of 50 Hz and a frame frequency of 25 fps
HDTV 480p -> Vertical frequency of 59.94Hz and a frame frequency of 29.97 fps
Other progressive modes ->  Vertical frequency of 60Hz and a frame frequency of 30 fps

Also progressive mode isn't a specific video mode, it's just a term used to describe all the video modes which have not set the INT flag.